quic/s2n-quic-crypto/src/negotiated.rs

```diff
@@ -100,15 +100,15 @@ macro_rules! negotiated_crypto {
         impl $name {
             /// Create a server cipher suite with a given negotiated algorithm and secret
             pub fn new_server(
-                algorithm: &$crate::Algorithm,
+                algorithm: &$crate::ring_aead::Algorithm,
```

Comment:
why is this necessary?

Reply:
See `BREAKING CHANGE` in the description. I removed some ring types we were exporting from this crate so now we need to access it directly from ring_aead